### ISSUE TITLE
Add OVR_multiview2 extension for Firefox iOS

### ISSUE THREAD

**Issue by ArtKoKo** [mozilla-mobile/firefox-ios#5693]
Is it possible to have the support of OVR_multiview2 on iOS like Firefox Desktop 71 ?
[https://github.com/mrdoob/three.js/pull/16316](url)

**Comment by garvankeeley** [comment]
Can you provide more info on ovr_multiview2 on what it is and why we need it, thanks

**Comment by ArtKoKo** [comment]
Yes I search to obtain the double View (with distortion) for a CardBoard with WebXR experiences.

**Comment by athomasmoz** [comment]
Thank you for logging. We are exploring options for WebXR in the future, I'll keep this open to track
